packages/styled-components/src/constructors/test/keyframes.test.js

```diff
@@ -7,12 +7,15 @@ import keyframes from '../keyframes';
 import Keyframes from '../../models/Keyframes';
 import { expectCSSMatches, getCSS, resetStyled } from '../../test/utils';
 
-/**
- * Setup
- */
+// Disable isStaticRules optimisation since we're not
+// testing for ComponentStyle specifics here
+jest.mock('../../utils/isStaticRules', () => () => false);
```

Comment:
In some files I'm disabling this so we minimise changes to the snapshot. I've only done it for files that would've had a lot of changes otherwise and that don't care about the `isStatic` optimisation